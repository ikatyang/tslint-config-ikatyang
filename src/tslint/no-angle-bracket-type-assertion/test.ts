// good

const good1 = 1 as number;

// bad

const bad1 = <number>1;
