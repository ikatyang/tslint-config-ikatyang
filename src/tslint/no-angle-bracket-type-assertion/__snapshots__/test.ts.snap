// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

const good1 = 1 as number;

// bad

const bad1 = <number>1;
             ~~~~~~~~~  [Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead.]

`;
